unison-src/transcripts/propagate.output.md

```diff
@@ -37,13 +37,13 @@ And then we add it.
 
 .subpath> find.verbose
 
-  1. -- #qae64o6am81hoadf7eabd909gojboi5iu3g9deip79ro18f11bbhir2vg51grg4m72kr5ikdovi6aupttet0nsqil7f0df9nqr10hqg
+  1. -- #fnvv5qd8qafiki64lnsuggdiuattl1tnso1la5ko2b25alvir0bt5jqpksj45qf3u1ckj7qpnps26rlq45dsvaeb98es5q4ike1fct8
```

Comment:
These GUIDs depend on the number of commands in the transcript before the point where the GUIDs are generated. So since we added some commands, they're obviously going to change. If we want transcript GUIDs to survive implementation changes, we need to implement the state machine that generates them some other way.